tests/run_all.sh

```diff
@@ -2,5 +2,11 @@
 
 export TEST_DIR="$PWD/tests"
 
-./main.sh -r . -c tests/img_test.in | diff tests/img_test.out -
-./main.sh -r . -c tests/base_test.in | diff tests/base_test.out -
+tests_to_run=(img_test imap_tests base_test)
+
+# Need splitting to occur to iterate over tests
+# shellcheck disable=SC2068
```

Comment:
Why is this being disabled here? Can't you add double quotes?

Reply:
You're right I just didn't know